docs/src/components/ReactExample/Example.tsx

```diff
@@ -18,14 +18,21 @@ const StyledWrapper = styled.div<{ isFullPage?: boolean }>`
     grid-template-columns: 1fr;
     border-radius: 12px;
     border: 1px solid ${theme.orbit.paletteCloudDark};
+    overflow: hidden;
   `};
 `;
 
-const StyledWrapperFrame = styled.div<{ width: number }>`
+const StyledWrapperFrame = styled.div<{ width: number | null }>`
   ${({ width }) => css`
     margin: 0 auto;
-    max-width: ${width}px;
     width: 100%;
+    ${width === null
```

Comment:
looks like it did not help, it's still broken after the hard refresh

Reply:
Fixed in another PR.